doc/tutorials/platformer/app/lib/actors/ember.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:ember_quest/actors/water_enemy.dart';
+import 'package:ember_quest/ember_quest.dart';
+import 'package:ember_quest/objects/ground_block.dart';
+import 'package:ember_quest/objects/platform_block.dart';
+import 'package:ember_quest/objects/star.dart';
+import 'package:flame/collisions.dart';
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flutter/services.dart';
+
+class EmberPlayer extends SpriteAnimationComponent
+    with KeyboardHandler, CollisionCallbacks, HasGameRef<EmberQuestGame> {
+  EmberPlayer({
+    required super.position,
+  }) : super(size: Vector2.all(64), anchor: Anchor.center);
+
+  final Vector2 velocity = Vector2.zero();
+  final Vector2 fromAbove = Vector2(0, -1);
+  final double gravity = 15;
+  final double jumpSpeed = 600;
+  final double moveSpeed = 200;
+  int horizontalDirection = 0;
+
+  bool hasJumped = false;
+  bool isOnGround = false;
+  bool hitByEnemy = false;
+
+  @override
+  Future<void> onLoad() async {
+    animation = SpriteAnimation.fromFrameData(
+      gameRef.images.fromCache('ember.png'),
+      SpriteAnimationData.sequenced(
+        amount: 4,
+        textureSize: Vector2.all(16),
+        stepTime: 0.12,
+      ),
+    );
+
+    add(
+      CircleHitbox()..collisionType = CollisionType.active,
```

Comment:
It is active by default so no need in specifying it

Reply:
Fixed.